lib/PostHog.php

```diff
@@ -210,10 +233,16 @@ private static function cleanHost(?string $host): string
     /**
      * Check the client.
      *
+     * @param bool $isPersonalApiKeyRequired
      * @throws Exception
      */
-    private static function checkClient()
-    {
+    private static function checkClient(
```

Comment:
I'm a bit confused by this 🤔  - why do we need `PersonalApiKey` for getting feature flags? I don't think this is a requirement for `/decide`

Reply:
Hm, don't we require a personal api key for all feature flag related api calls? It seems like we require this for all other libraries (see `posthog-go` [here](https://posthog.com/docs/integrate/server/go#feature-flags)) and we were missing this check in our php library.

Reply:
.. and taking another look, php doesn't call `api/feature_flag` at all, does it?

Reply:
Thanks for the context. Nope we don't call api/feature_flag anywhere. This makes my life a lot easier thanks

Reply:
Quick flyby that some places do need that check in go: https://github.com/PostHog/posthog-go/blob/master/featureflags.go#L97 - but haven't looked deeper into where exactly. Thanks for doing this!